wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/de.md

```diff
@@ -0,0 +1,66 @@
+# Befehle für das Turnier-Management
+
+Die folgenden Chatbefehle sind für die Fernverwaltung von Mehrspieler-Turnierräumen vorgesehen:
+
+- `!mp make <name>` - Erstellt einen Turnierraum mit dem angegebenen Namen. Maximal 4 solcher Räume können erstellt werden.
+  - Dieser Raum ist insofern besonders, da er nicht geschlossen wird, sobald alle Spieler den Raum verlassen haben. Außerdem ist er passwortgeschützt vor Spielern, die ihm selbst beitreten wollen.
```

Comment:
```suggestion
  - Dieser Raum ist insofern besonders, da er nicht geschlossen wird, sobald alle Spieler den Raum verlassen haben. Außerdem ist er vor Spielern, die ihm selbst beitreten wollen, passwortgeschützt.
```

Reply:
I try to minimise the usage of commas as it generally improves readability, but I'm fine with this suggestion.